README.md

```diff
@@ -167,6 +167,19 @@ The response object for the same request in earlier versions than to 2.0.0.
   }
 ```
 
+To support making API calls with specific headers you may pass in the desired headers under the `headers` key for any
+wrapper call that supports optional parameters; e.g. getting a playlist could have usage
+```javascript
+spotifyApi.getPlaylist('odesza','0uxUkSQHZVM7so7msXl9Ck', { headers: { 'If-None-Match': 'AAAANPBn03qInch471K02Rncz5IHWVja'} })
+  .then(function(data) {
+    if(data.statusCode !== 304)
```

Comment:
Minor formatting comment, but if and parentheses should have a space between them according to [Code Conventions for the JavaScript Programming Language](http://javascript.crockford.com/code.html).